plugins/inventory/kubevirt.py

```diff
@@ -16,23 +16,23 @@
 - "KubeVirt.io Project (!UNKNOWN)"
 
 description:
-- Fetch running VirtualMachineInstances for one or more namespaces with an optional label selector.
-- Groups by namespace, namespace_vmis and labels.
-- Uses the kubectl connection plugin to access the Kubernetes cluster.
-- Uses *.kubevirt.(yml|yaml) YAML configuration file to set parameter values.
+- Fetch running C(VirtualMachineInstances) for one or more namespaces with an optional label selector.
+- Groups by cluster name, namespace and labels.
+- Uses the M(kubernetes.core.kubectl) connection plugin to access the Kubernetes cluster.
+- Uses V(*.kubevirt.[yml|yaml]) YAML configuration file to set parameter values.
 
 extends_documentation_fragment:
 - inventory_cache
 - constructed
 
 options:
   plugin:
-    description: Token that ensures this is a source file for the "kubevirt" plugin.
+    description: Token that ensures this is a source file for the M(kubevirt.core.kubevirt) plugin.
```

Comment:
This links to the `kubevirt.core.kubevirt` module. If you want to link to the kubevirt inventory plugin, use `P`:
```suggestion
    description: Token that ensures this is a source file for the P(kubevirt.core.kubevirt#inventory) plugin.
```

Reply:
Updated, thanks!